lib/Jit/LLILCJit.cpp

```diff
@@ -303,6 +283,16 @@ CorJitResult LLILCJit::compileMethod(ICorJitInfo *JitInfo,
 
       // Tell the CLR that we've successfully generated code for this method.
       Result = CORJIT_OK;
+
+      if (Context.Options->DumpLevel >= ::DumpLevel::SUMMARY) {
+         errs() << "LLILC compiled: " 
+                << ", Entry = " << *NativeEntry 
+                << ", End = " << (*NativeEntry + *NativeSizeOfCode)
+                << ", size = " << *NativeSizeOfCode 
+                << " method = " << Context.MethodName 
+                << '\n';
+      }
+
```

Comment:
Touch base with @erozenfeld,  The semantics of this are now a little different than the old check.  We're not marking successful reading, but additionally the full compile.